src/pipeline.py

```diff
@@ -86,12 +86,30 @@ def run_pipeline(start, config_path):
     # Data Ingest
     MainLogger.info("Starting Data Ingest...")
 
+    # NI staging and validation
```

Comment:
Is this supposed to be in this ticket?

Reply:
I think I might have just added this in for consistency in the comments.

Reply:
NI staging is being run twice in `pipeline.py` - a temp run here at line 90-106 (just staging) and again at line 142-145 (full staging and construction).  Should this bit be taken out now?